tests/static/app_registry.yaml

```diff
@@ -15,7 +15,7 @@ releases:
     v22.11.0:
         environment:
             python_requirements:
-                - Jinja2~=2.11.3
+                - 'Jinja2~=2.11.3 # invalid requirement'
```

Comment:
Does the test check this?

Reply:
Thanks, good point. The test checked this implicitly since previously this would cause an exception. I've added an explicit unit test in `test_appdataclass.py`.